cmd/ocsp-responder/main_test.go

```diff
@@ -34,6 +34,18 @@ var (
 	stats = metrics.NoopRegisterer
 )
 
+func mustRead(path string) []byte {
+	f, err := os.Open(path)
+	if err != nil {
+		panic(fmt.Sprintf("open %#v: %s", path, err))
+	}
+	b, err := ioutil.ReadAll(f)
```

Comment:
You can simplify this function with `ioutil.ReadFile`

Reply:
Done! I just moved this up here because it was defined way down in the middle of the file despite being a utility relied upon by the very next function. But this is a good cleanup!